src/LanguageServer.php

```diff
@@ -100,4 +120,42 @@ public function exit()
     {
         exit(0);
     }
+
+    /**
+     * Parses workspace files, one at a time.
+     *
+     * @param string $rootPath The rootPath of the workspace. Is null if no folder is open.
+     * @return void
+     */
+    private function indexProject(string $rootPath)
```

Comment:
Shouldn't we check this at the calling site? Indexing a null path doesn't make sense, after all.